### ISSUE TITLE
Hide icons between tags?

### ISSUE THREAD

**Issue by hbhargava7** [dwarvesf/hidden#263]
Is it possible to make icons between selectors collapsible, but anything to left OR right of selection persists?

e.g. expanded: `A B * C D E * F G`
and collapsed: `A B * F G `

My use case is that the webcam notch on newer Macbooks hides the leftmost items, so hiding items in the middle or on the right would make space and reveal them.

Is this possible? Thanks for maintaining this tool!

**Comment by UeharaYou** [comment]
No, as far as I concerned. The implementation of Hidden Bar simply cannot accomplish that via APIs it's using. Though the functionality itself is possible, it requires using another set of APIs, which currently not a feature of Hidden Bar.
